### PR TITLE
Fat test to check if app using metrics is held in memory after unloading said app

### LINKED ISSUE #2866: Add a test to ensure that an undeployed app with metrics isn't held in memory by mpMetrics-1.0/1.1

### PR DESCRIPTION
**Pull request by Channyboy**
#build 
fixes #2866

### CODE REVIEW

**Comment by Channyboy** [comment]
#build

**Comment by Channyboy** [comment]
#build

File: dev/com.ibm.ws.microprofile.metrics_fat/fat/src/com/ibm/ws/microprofile/metrics/tck/launcher/MetricsClassLoaderTest.java [hunk 1]
```diff
@@ -0,0 +1,114 @@
...
+                break;
+        }
+        /*
+         * checkServlet returns a classloader's (string) value or "null"
+         * We want it to be "null" now
+         */
+        Assert.assertFalse("variable: `classLoaderAfterupdate` shouldn't be a null", classLoaderAfterUpdate == null);
+
+        //In the event of classloader not being unloaded, we will create a javacore, heap dump  and thread dump.
+        if (!classLoaderAfterUpdate.equals("null")) {
```
Comment (fmhwong) [inline dev/com.ibm.ws.microprofile.metrics_fat/fat/src/com/ibm/ws/microprofile/metrics/tck/launcher/MetricsClassLoaderTest.java]: Use NULL_STRING?

File: dev/com.ibm.ws.microprofile.metrics_fat/fat/src/com/ibm/ws/microprofile/metrics/tck/launcher/MetricsClassLoaderTest.java [hunk 2]
```diff
@@ -0,0 +1,114 @@
...
+            getServer().javadumpThreads();
+            getServer().serverDump("heap, system");
+        }
+
+        Assert.assertTrue("ClassLoader still in memory; App not properly unloaded variable `classLoaderAfterupdate` = " + classLoaderAfterUpdate,
+                          classLoaderAfterUpdate.equals("null"));
```
Comment (fmhwong) [inline dev/com.ibm.ws.microprofile.metrics_fat/fat/src/com/ibm/ws/microprofile/metrics/tck/launcher/MetricsClassLoaderTest.java]: Use NULL_STRING?

**Comment by Channyboy** [comment]
#build

**Comment by Channyboy** [comment]
#build

**Pull request merged**